tools/build.iosReleaseBuild.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Please remove this file from the commit.

Reply:
Fixed: Files are remove.